test/integration/uk/gov/hmrc/exports/repositories/SubmissionRepositorySpec.scala

```diff
@@ -22,16 +22,16 @@ import play.api.inject.guice.GuiceApplicationBuilder
 import uk.gov.hmrc.exports.models.{CancellationRequestExists, CancellationRequested, MissingDeclaration, Submission}
 import uk.gov.hmrc.exports.repositories.SubmissionRepository
 import unit.uk.gov.hmrc.exports.base.CustomsExportsBaseSpec
-import util.ExportsTestData
 
-class SubmissionRepositorySpec extends CustomsExportsBaseSpec with ExportsTestData with BeforeAndAfterEach {
+class SubmissionRepositorySpec extends CustomsExportsBaseSpec with BeforeAndAfterEach {
+  import SubmissionRepositorySpec._
 
   override lazy val app: Application = GuiceApplicationBuilder().build()
-  private val repo: SubmissionRepository = component[SubmissionRepository]
+  private val repo: SubmissionRepository = app.injector.instanceOf[SubmissionRepository]
```

Comment:
👍 Good to remove usage of `component`

We should get rid of this method in general, it makes the code less readable